docs/install/knative-with-operators.md

```diff
@@ -1,84 +1,74 @@
 ---
-title: "Installing Knative components using Operator"
-weight: 10
+title: "Knative Operator installation"
+weight: 02
 type: "docs"
+showlandingtoc: "false"
 ---
 
-Knative provides an [operator](https://github.com/knative/operator) as a tool to install, configure and manage Knative. The Knative operator leverages custom objects
-in the cluster to define and manage the installed Knative software. This guide explains how to install and uninstall
-Knative using Knative operator.
+Knative provides a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to install, configure and manage Knative.
 
-## Before you begin
-
-Knative installation using the Operator requires the following:
-
-- A Kubernetes cluster v1.16 or newer, as well as a compatible kubectl. This guide assumes that you've already created
-a Kubernetes cluster. If you have only one node for your cluster, set CPUs to at least 6, Memory to at least 6.0 GB,
-Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set CPUs to at least 2, Memory to at least
-4.0 GB, Disk storage to at least 20 GB for each node.
-- The Kubernetes cluster must be able to access the internet, since the Knative operator downloads images online.
-- [Download and install Istio](./installing-istio.md).
-
-## Limitations of Knative Operator:
-
-Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
+**NOTE:** The Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
 for development or test purposes only.
 
-## Install Knative with the Knative Operator
+## Prerequisites
 
-You can find the release information of Knative Operator on the [Releases page](https://github.com/knative/operator/releases).
+- You have a cluster that uses Kubernetes v1.17 or newer.
+- You have installed the `kubectl` CLI.
+- If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
+- If you have multiple nodes in your cluster, for each node you will need at least 2 CPUs, 4 GB of memory, and 20 GB of disk storage.
+- Your Kubernetes cluster must have access to the internet, since the Knative Operator must download images.
```

Comment:
I think the YAML version requires the same thing -- I don't think the Operator is downloading images directly anymore, but Kubernetes will still need to fetch images.

@houshengbo to confirm

Reply:
It should be phrased in different way the cluster doesn’t need access to the internet but it needs access to the container registry that the referenced images are stored. 
this is because admin has the choice to do airgap installations by copying g the images to internal registry and modify the default image references

Reply:
@evankanderson Right, kube still needs to fetch images, like gcr.io/knative-releases/knative.dev/operator/cmd/operator:v0.21.0.
The operator itself does not.

Reply:
> this is because admin has the choice to do airgap installations by copying g the images to internal registry and modify the default image references

@csantanapr is the air gapped installation documented somewhere? Otherwise I will leave this as "access to the internet" because mentioning this opens a can of worms that I don't have time to dig into documenting right now. Can you please open an issue in the docs repo for air gapped installs if it's not documented or tracked yet?

@houshengbo thanks for confirming, I'm following up the comments in another PR and will update this to say kube instead of the Operator.